Competition2017/src/competition/subsystems/vision/VisionSubsystem.java

```diff
@@ -111,11 +118,13 @@ private void handleCommPacket(OffboardCommPacket packet) {
 
     @Override
     public void updatePeriodicData() {
-        double currentTime = Timer.getFPGATimestamp();
-        if (currentTime - lastTimePacketRecieved >= 5 || packet == null) {
+        if (packetsPerSecond >= timeThreshold.get() || lastConnectionState) {
             log.info("Disconnected");
         } else {
-            log.info(packet.id + " (" + packet.packetType + "): " + packet.payload.toString());
+            log.info("Connected");
+            packetsPerSecond = numberOfPackets / lastTimePacketRecieved; 
```

Comment:
The "connected" log should only run once a match; you want this math and FPS logging to happen on an interval constantly.